tests/test_views.py

```diff
@@ -184,11 +190,15 @@ def test_correct_template_used(self):
         response = self.client.get('/withers/location/United_States/')
         self.assertTemplateUsed(response, 'citeIt/index.html')
 
-#   def test_query_matching_locations(self):
-#       # Create ten citations to match the query and one that doesn't.
-#       hold = factories.CitationFactory.create_batch(10)
-#       response = self.client.get('/withers/location/United_States/')
-#       self.assertEqual(len(response.context['citations']), 10)
+    def test_query_matching_locations(self):
+        # Create ten citations to match the query and one that doesn't.
+        location_us = factories.LocationFactory(location='United States')
+        location_mexico = factories.LocationFactory(location='Mexico')
+        factories.CitationFactory.create_batch(10, coverage=(location_us, ))
+        factories.CitationFactory(coverage=(location_mexico, ))
+        response = self.client.get('/withers/location/United_States/')
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(response.context['citations']), 10)
```

Comment:
There are couple of tests in `test_views.py` like this one (not many). @ldko and I agree that they are difficult to read. What do you think about creating some logical groupings with some whitespace?

For example,
```python
     def test_query_matching_locations(self):
       # Create ten citations to match the query and one that doesn't.
        location_us = factories.LocationFactory(location='United States')
        location_mexico = factories.LocationFactory(location='Mexico')

        factories.CitationFactory.create_batch(10, coverage=(location_us, ))
        factories.CitationFactory(coverage=(location_mexico, ))

        response = self.client.get('/withers/location/United_States/')
        self.assertEqual(response.status_code, 200)
        self.assertEqual(len(response.context['citations']), 10)
````

I find this a little easier to read. 
    1. setup the locations
    2. setup the factories
    3. call and assert

This can't and shouldn't be applied to all tests, but I think it works in this scenario.